great_tables/_formats.py

```diff
@@ -2593,7 +2601,7 @@ def _get_locale_sep_mark(default: str, use_seps: bool, locale: Union[str, None]
     # the column named 'group'; this could potentially be of any type but we expect
     # it to be a string (and we'll check for that here)
     sep_mark: Any
-    sep_mark = pd_df_row.iloc[0]["group"]
+    sep_mark = pd_df_row["group"]
```

Comment:
After filtering DataFrames to be a 1 row DataFrame, a specific value was peeled off. Now it's just pulled out of the row dictionary.